tests/python/relay/test_pass_fake_quantization_to_integer.py

```diff
@@ -303,6 +303,19 @@ def test_fake_quantize_global_avg_pool():
     compare_fq_to_int(op, [x_np], True)
 
 
+def test_fake_quantize_rsqrt():
+    x = relay.var("x", shape=[1, 3, 224, 224], dtype="int8")
+    zero = relay.const(0)
+
+    x = relay.qnn.op.dequantize(x, relay.const(2.0), zero)
+    op = relay.rsqrt(x)
+    op = relay.qnn.op.quantize(op, relay.const(2.0), zero)
+
+    x_np = np.random.randint(-128, 127, size=[1, 3, 224, 224], dtype="int8")
+
+    compare_fq_to_int(op, [x_np], True)
```

Comment:
Huh, shouldn't this fail? since the default lowering matches the fake-quantized version?

Reply:
It lowers to qnn.rsqrt

```
def @main(%x: Tensor[(1, 3, 224, 224), int8]) -> Tensor[(1, 3, 224, 224), int8] {
  %0 = qnn.dequantize(%x, 2f /* ty=float32 */, 0 /* ty=int32 */, Tensor[(1, 3, 224, 224), int8], float32, int32) /* ty=Tensor[(1, 3, 224, 224), float32] */;
  %1 = rsqrt(%0, Tensor[(1, 3, 224, 224), float32]) /* ty=Tensor[(1, 3, 224, 224), float32] */;
  qnn.quantize(%1, 2f /* ty=float32 */, 0 /* ty=int32 */, Tensor[(1, 3, 224, 224), float32], float32, int32, out_dtype="int8") /* ty=Tensor[(1, 3, 224, 224), int8] */
}
```
vs.
```
 def @main(%x: Tensor[(1, 3, 224, 224), int8]) -> Tensor[(1, 3, 224, 224), int8] {
  qnn.rsqrt(%x, 2f /* ty=float32 */, 0 /* ty=int32 */, 2f /* ty=float32 */, 0 /* ty=int32 */, Tensor[(1, 3, 224, 224), int8], float32, int32, float32, int32) /* ty=Tensor[(1, 3, 224, 224), int8] */
}
```

Reply:
Oh yeah, this is canonicalized later makes sense